projects/plugins/jetpack/_inc/client/components/connect-button/index.jsx

```diff
@@ -293,8 +324,8 @@ export default connect(
 			fetchSiteConnectionStatus: () => {
 				return dispatch( fetchSiteConnectionStatus() );
 			},
-			unlinkUser: () => {
-				return dispatch( unlinkUser() );
+			unlinkUser: ( isOwner = false ) => {
+				return dispatch( unlinkUser( isOwner ) );
```

Comment:
It appears that `isOwner` argument is never passed to this function, neither it's accepted by the `unlinkUser()` state action.

Reply:
Good catch! Fixed it.